src/beginner-setup.md

```diff
@@ -0,0 +1,109 @@
+# Beginner Setup Guide: Running Sugar Activities Locally
+
+This guide will help beginners set up a local development environment for Sugar and run Sugar activities easily.
+
+---
+
+## 🧰 Prerequisites
+
+Before starting, make sure you have the following tools installed:
+
+### For Linux Users (Ubuntu/Debian-based):
+
+* `git`
+* `python3`
+* `pip`
+* `virtualenv`
```

Comment:
Why do you think pip and virtualenv are required?  They are not.  We target distributions, not ad-hoc virtual environments.  Any guidance that suggests development inside a Python virtual environment is going to mislead developers.  Python virtual environments may be used at a pinch, but testing must involve running Sugar and the activity in native mode.